charts/zookeeper-operator/templates/operator.yaml

```
@@ -21,6 +21,13 @@ spec:
       {{- end }}
     spec:
       serviceAccountName: {{ .Values.serviceAccount.name }}
+      volumes:
+{{- if .Values.logForward.create }}
+{{- include "fluent-bit-sidecar-volume" . | indent 6 }}
+{{- end }}
```

remove the .create flag and use just the template. no fluent-bit reference
```suggestion
{{- include "chart.AdditionalVolumes" . | indent 6 }}
```

---

chart.AdditionalVolumes added.